go.mod

```diff
@@ -12,7 +12,6 @@ replace (
 	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc => go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.58.0
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.58.0
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc => go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.33.0
-	golang.org/x/oauth2 => golang.org/x/oauth2 v0.25.0
```

Comment:
Is it no longer needed to ping it to a specific tag?  I noticed the auto-generated PR bumps the tag to  v0.27.0

Reply:
The [indirect dependency](https://github.com/rancher/wins/blob/main/go.mod#L160C2-L160C29) was already pointing to `v0.27.0`, and ideally this would always match what is being used by the system-agent.

But I don't mind keeping the explicit version pinning, I've added the replace statement back.